person.rb

```diff
@@ -0,0 +1,26 @@
+require('securerandom')
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(name = 'Unknown', age = nil, parent_permission = nil)
+    @id = SecureRandom.uuid
+    @name = name
```

Comment:
- [ ] Good job setting default value for optional variable 👍 , As for the `age` variable , it should not be defined with a default value as it is not a optional variable . Note that default value should defined as  "Unknown" for name and true for parent_permission  , Kindly fix that 👍 

![Capture](https://github.com/wacoo/school-library-system/assets/80458199/3570d395-053b-40fc-9190-b020a5a14ea1)

```ruby
class Person
.
.
.
def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end
.
.
.
end
```